modelforge/potential/schnet.py

```diff
@@ -237,7 +244,7 @@ def forward(self, pairlist: Dict[str, torch.Tensor]) -> Dict[str, torch.Tensor]:
 
         idx_i, idx_j = atom_index12[0], atom_index12[1]
 
-        return x
+        return {"f_ij": f_ij, "idx_i": idx_i, "idx_j": idx_j, "rcut_ij": rcut_ij}
 
 
 class LighningSchnet(Schnet, LighningModuleMixin):
```

Comment:
Two things here:

- We should probably consistent of `Schnet` vs `SchNet`
- minor typo in Lightning (forgot the t).